### ISSUE TITLE
ScaffoldWalk Wurst bypass

### ISSUE THREAD

**Issue by dani02021** [HorizonCode/PNCP#19]
Video: https://youtu.be/6pPEIuLQte8

**Comment by BlackHawk2306** [comment]
It isn't really patchable without alot of false postitives, because wihtout sprinting, you walk backwards on server side when using scaffold.

**Comment by dani02021** [comment]
Nothing is impossible ;)

**Comment by BlackHawk2306** [comment]
Yep, but especially on german servers, where people are sweating more, some can build without sneaking (breezily bridge i.e.)

**Comment by dani02021** [comment]
It's true that there could be false positives, depends on the check, you don't need to check if they are sneeking to place a block, there are other ways also. Some are just copy/pasting some bad code so you can very easy catch the check, Wurst is not using really good scaffold I can say
